### PR TITLE
Remove more references to grit and start updating readme

### PR DESCRIPTION
**Pull request by dometto**
I removed the Videos section from the Readme because the videos are all based on 4.x. Or is it better to have something rather than nothing here?

### CODE REVIEW

**Comment by dometto** [comment]
@bartkamphorst I've updated the readme and wiki to link to some of the new features in 5.x. Could you have a link and see if there's anything else we should mention either in this Readme, or that's currently in the 5.0 Release Notes, but should have a different place in the wiki? Almost there I think!

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ gollum -- A git-based Wiki
 [![Build Status](https://travis-ci.org/gollum/gollum.svg?branch=master)](https://travis-ci.org/gollum/gollum)
 [![Dependency Status](https://gemnasium.com/gollum/gollum.svg)](https://gemnasium.com/gollum/gollum)
 
-Note: this is the development branch for the next major release of Gollum, version 5.0. See [here](https://github.com/gollum/gollum/wiki/5.0-release-notes) for a list of changes and new features.
+**Gollum version 5.0 is out!** See [here](https://github.com/gollum/gollum/wiki/5.0-release-notes) for a list of changes and new features compared to the 4.x branch.
```
Comment (bartkamphorst) [inline README.md]: I wouldn't necessarily refer to versions 4.x as a 'branch'. Just `features as compared to gollum version 4.x` or something like that.

File: README.md [hunk 2]
```diff
@@ -15,34 +15,17 @@ Gollum is a simple wiki system built on top of Git. A Gollum Wiki is simply a gi
 	* May be written in a variety of [markups](#markups).
 	* Can be edited with your favourite system editor or IDE (changes will be visible after committing) or with the built-in web interface.
 	* Can be displayed in all versions (commits).
+* Gollum supports advanced functionality like:
+  * [UML diagrams](https://github.com/gollum/gollum/wiki#plantuml-diagrams)
+  * [BibTeX and Citation support](https://github.com/gollum/gollum/wiki/BibTeX-and-Citations)
+  * Annotations using [CriticMarkup](https://github.com/gollum/gollum/wiki#criticmarkup-annotations)
```
Comment (bartkamphorst) [inline README.md]: CriticMarkup in the wiki could do with a screenshot.

File: README.md [hunk 3]
```diff
@@ -15,34 +15,17 @@ Gollum is a simple wiki system built on top of Git. A Gollum Wiki is simply a gi
...
+  * Mathematics via [MathJax](https://github.com/gollum/gollum/wiki#mathematics)
```
Comment (bartkamphorst) [inline README.md]: Mathematics can also do with a screenshot.

File: README.md [hunk 4]
```diff
@@ -54,11 +37,17 @@ Varies depending on operating system, package manager and Ruby installation. Gen
 	[sudo] gem install gollum
 	```
 
-Installation examples for individual systems can be seen [here](https://github.com/gollum/gollum/wiki/Installation).
+To run, simply:
+
+1. Navigate to your git repository (wiki) via the command line.
+2. Run: `gollum`.
+3. Open `http://127.0.0.1:4567` in your browser.
```
Comment (bartkamphorst) [inline README.md]: Perhaps `http://localhost:4567` better?

File: README.md [hunk 5]
```diff
@@ -54,11 +37,17 @@ Varies depending on operating system, package manager and Ruby installation. Gen
...
+
+Of course, you can also run `gollum /path/to/wiki`. Installation examples for individual systems can be seen [here](https://github.com/gollum/gollum/wiki/Installation).
```
Comment (bartkamphorst) [inline README.md]: Perhaps this should be the default recommendation. When I started using gollum I was annoyed to find that all my pages had cluttered my gollum install.

File: README.md [hunk 6]
```diff
@@ -54,11 +37,17 @@ Varies depending on operating system, package manager and Ruby installation. Gen
...
 
 ### Markups
 
-Gollum presently ships with support for the following markups:
+Gollum allows using different markup languages per page. It presently ships with support for the following markups:
```
Comment (bartkamphorst) [inline README.md]: This is ambiguous and suggests that you can mix different markups on a single page.

File: README.md [hunk 7]
```diff
@@ -79,24 +68,13 @@ By default, Gollum ships with the `kramdown` gem to render Markdown. However, yo
...
+## RUNNING FROM SOURCE
 
 1. `git clone https://github.com/gollum/gollum`
 2. `cd gollum`
-3. `[sudo] bundle install` (may not always be necessary).
+3. `[sudo] bundle install`
 4. `bundle exec bin/gollum`
-	* Like that, gollum assumes the target wiki (git repository) is the project repository itself. If it's not, execute `bundle exec bin/gollum <path-to-wiki>` instead.
-5. Open `http://localhost:4567` in your browser.
+5. Open `http://127.0.0.1:4567` in your browser.
```
Comment (bartkamphorst) [inline README.md]: Perhaps `http://localhost:4567` is better?

File: README.md [hunk 8]
```diff
@@ -120,44 +98,47 @@ Gollum comes with the following command line options:
...
+| --versions        | none      | Display the current version of Gollum and auxiliary gems. |
 
 **Notes:**
 
 1. The `0.0.0.0` IP address allows remote access. Should you wish for Gollum to turn into a personal Wiki, use `127.0.0.1`.
 2. Before using `--adapter`, you should probably read [this](https://github.com/gollum/gollum/wiki/Git-adapters) first.
 3. When `--css` or `--js` is used, respective files must be committed to your git repository or you will get a 302 redirect to the create a page.
-4. Files can be uploaded simply by dragging and dropping them onto the editor's text area (this is, however exclusive to the default editor, not the live preview editor).
-5. Read the relevant [Security note](https://github.com/gollum/gollum/wiki/Security#custom-cssjs--mathjax-config) before using these.
+4. Files can be uploaded simply by dragging and dropping them onto the editor's text area.
```
Comment (bartkamphorst) [inline README.md]: Only if `--allow-uploads` is set to true.

File: README.md (hunk 2 above)
Comment (dometto) [reply README.md]: This now has its own issue

**Comment by dometto** [comment]
Thanks for the feedback 👍

**Comment by dometto** [comment]
@bartkamphorst I think I've added the features from the 5.0 release notes that need their own section on the main page of the wiki, to the main page of the wiki. :) Please have a look if you think anything is missing there.

Also: some features, like editable sections, don't really require documentation. But it would be nice to advertise them a little. Can you think of a good way to do that? Perhaps it would just be best to have a live instance of gollum for that.

**Pull request merged**